test/gtest/cfg.cpp

```diff
@@ -65,7 +67,7 @@ TEST_F(CFGTest, Print) {
 ;; preds: [3], succs: [6]
 4:
   6: i32.const 3
-  7: block
+  7: block (result i32)
```

Comment:
Unrelated change?

Reply:
No, this is a necessary change. Probably due to a difference in finalization between the parsing for `block` and the parsing for `then`, but not sure it's worth digging into more.

Reply:
Makes sense, it's probably that.